src/Adapter/Platform/Oracle.php

```diff
@@ -9,12 +9,22 @@
 
 namespace Zend\Db\Adapter\Platform;
 
+use Zend\Db\Adapter\Exception;
+use Zend\Db\Adapter\Driver\DriverInterface;
+use Zend\Db\Adapter\Driver\Oci8;
+use Zend\Db\Adapter\Driver\Pdo;
+
 class Oracle extends AbstractPlatform
 {
+
+    /** @var resource|\PDO */
```

Comment:
resource is not a type allowed in constructor neither setDriver. Also types mismatch with the assigned in setDriver